### ISSUE TITLE
create sequence diagram for conneg-by-ap process

### ISSUE THREAD

**Issue by rob-metalinkage** [w3c/dxwg#852]
As per action-312.

**Comment by nicholascar** [comment]
Dependent on Issue https://github.com/w3c/dxwg/issues/603

**Comment by nicholascar** [comment]
Determined this issue is not necessary due to the simplicity of all Client/Server interaction

**Comment by aisaac** [comment]
@nicholascar is there a reference to a discussion on this?

For the record I'm ready to support a decision to "close-won't fix" for this issue. I would have like to see diagrams like for example https://www.w3.org/TR/swbp-vocab-pub/#recipe4 but I reckon that the interactions in this spec are simple enough, indeed!

**Comment by kcoyle** [comment]
@aisaac Please re-open this if you think there is more to discuss - otherwise we could lose track of it within the closed issues. The only way we have to keep track is to keep issues open (or create a label for closed issues that are questioned).

**Comment by aisaac** [comment]
I don't think there is need to re-open. As said I can support the resolution. It's just that I was wondering whether there was a reference for @nicholascar 's point.
